contracts/v0.8/utils/Actor.sol

```diff
@@ -72,9 +72,6 @@ library Actor {
     }
 
     function getErrorCodeMsg(int256 exit_code) internal pure returns (string memory) {
-        if (exit_code < 0) {
```

Comment:
The exit_code can be negative or positive, so we need to add the - if the value is neg. Openzeppelin has not support to convert int256 to string yet, and that is why we are applying that conditional @NufailIsmath

Reply:
Thanks for the advice. I see we have an error there. The idea was this one. I think we can use the abi.encodePacked instead of the string.concat. It is more performant [link](https://ethereum.stackexchange.com/questions/729/how-to-concatenate-strings-in-solidity).

```
function getErrorCodeMsg(int256 exit_code) internal pure returns (string memory) {
        if (exit_code < 0) {
            return string.concat("actor error code -", Strings.toString(uint256(exit_code)));
        }
        return string.concat("actor error code ", Strings.toString(uint256(exit_code)));
    }
```

Reply:
Cool then I'll correct it and do a pull request?